src/utils.rs

```diff
@@ -81,9 +81,9 @@ pub fn snippet_block<'a>(cx: &Context, span: Span, default: &'a str) -> Cow<'a,
 /// Trim indentation from a multiline string
 /// with possibility of ignoring the first line
 pub fn trim_multiline(s: Cow<str>, ignore_first: bool) -> Cow<str> {
-    let s = trim_multiline_inner(s, ignore_first, ' ');
-    let s = trim_multiline_inner(s, ignore_first, '\t');
-    trim_multiline_inner(s, ignore_first, ' ')
+    let s_space = trim_multiline_inner(s, ignore_first, ' ');
```

Comment:
I agree that this change is not ideal. But I also didn't want to slap `#[allow(shadow_reuse)]` on the function.

As for the lint group, perhaps we shall add a `clippy_all` group that includes all lints, and remove the `allow`ed lints from `clippy`.